app.js

```diff
@@ -28,7 +14,6 @@ var express = require('express'),
 var favicon = require('serve-favicon'),
     cookieSession = require('cookie-session'),
     cookieParser = require('cookie-parser'),
-    logger = require('morgan'),
```

Comment:
Are there any alternative logging mechanisms that you're using? I think it's a pretty bad idea to get rid of *all* console logs.